src/main.c

```diff
@@ -43,17 +43,39 @@
 
 #include <contiki.h>
 
+/* The two includes below are only needed for the example code */
+#include "sys/clock.h"
+#include "dev/leds.h"
+
 PROCESS(main_process, "Main process");
 AUTOSTART_PROCESSES(&main_process);
 /*---------------------------------------------------------------------------*/
 PROCESS_THREAD(main_process, ev, data)
 {
-  PROCESS_BEGIN();
+    PROCESS_BEGIN();
```

Comment:
Contiki's style is to give two spaces for indentation.
Please replace tabs with two spaces in all PR's.